eve/main.yml

```diff
@@ -92,7 +92,7 @@ models:
       env: *global-env
   - ShellCommand: &yarn-install
       name: install modules
-      command: yarn install --ignore-engines --frozen-lockfile --network-concurrency 1
+      command: yarn install --ignore-engines --frozen-lockfile --network-concurrency=1
```

Comment:
here should we use `=` or space?

Reply:
I am guessing both are valid, but as this was changed ion 7.10, to stay consistent, I wanted to keep this change while cherry-picking